qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/datamodel/resolvers/ValueResolverInfo.java

```diff
@@ -32,6 +32,10 @@ public class ValueResolverInfo {
 	private String signature;
 
 	private String sourceType;
+	
+	private Boolean sourceTypeSource;

```

Comment:
I find this name is ambigous with sourceType. I would use binary as field and to be consistent with ResolvedJavaType I will change isSource with isBinary by using binary field instead of using source field